crates/oxc_linter/src/snapshots/eslint_max_lines.snap

```diff
@@ -34,7 +33,7 @@ snapshot_kind: text
    ╰────
   help: Maximum allowed is 2.
 
-  ⚠ eslint(max-lines): File has too many lines (8).
+  ⚠ eslint(max-lines): File has too many lines (4).
```

Comment:
this doesn't look correct? the diagnostic message is saying that the file has 4 lines, but the diagnostic pointer (L39 of the snapshot) it pointing to L8?

Reply:
oh i see is this something todo with comment/blank lines?

Reply:
Pointer use last string for diagnostic, but calculation not includes black lines and comments if they true